pkg/pwalkdir/pwalkdir.go

```diff
@@ -0,0 +1,105 @@
+// +build go1.16
+
+package pwalkdir
+
+import (
+	"fmt"
+	"io/fs"
+	"path/filepath"
+	"runtime"
+	"sync"
+)
+
+type WalkFunc = fs.WalkDirFunc
```

Comment:
Probably want to rename the alias as well. `WalkDirFunc` is not a drop-in replacement for `pwalk.WalkFunc` (different signature), so perhaps better to rename?

```suggestion
type WalkDirFunc = fs.WalkDirFunc
```

Reply:
Or should we remove the alias altogether? The signature has `fs.Direntry` in it, so consumers would still have to add `import "io/fs"` (so no extra convenience there)

Reply:
Yep, I think it is best to remove the alias.

Reply:
removed